coj/4265.cpp

```diff
@@ -1,4 +1,5 @@
-#include <bits/stdc++.h>
+// Problem link: https://coj.uci.cu/24h/problem.xhtml?pid=4265
+
#include <bits/stdc++.h>
```

Comment:
many files have this weird character, should remove it

Reply:
solved